sklearn/metrics/cluster/tests/test_unsupervised.py

```diff
@@ -68,3 +68,64 @@ def test_correct_labelsize():
                          'Number of labels is %d\. Valid values are 2 '
                          'to n_samples - 1 \(inclusive\)' % len(np.unique(y)),
                          silhouette_score, X, y)
+
+
+def test_paper_example():
```

Comment:
test_paper_example is not a great name for a test :P . Would be great to mention the name of the paper.